README.md

```diff
@@ -68,16 +68,16 @@ It's dangerous to go alone! Take these:
 
 Command | Summary
 :- | :-
-`yarn storybook` | Develop components locally using Storybook
-`STORIES=path/to/package yarn storybook` | Develop just one package at a time using Storybook (builds faster!)
-`yarn commit` | Use commitizen to help you write your conventional commits
-`yarn compile` | Run all typechecks
-`yarn lint` | Run all linters
-`yarn lint:fix` | Run all linters, fixing violations
-`yarn plop` | Add a new component/subcomponent
-`yarn test:storybook` | Run tests on all Storybook stories
-`yarn test` | Run all [Jest](https://jestjs.io/) tests
-`yarn reset` | Reinstall all dependencies
+`pnpm storybook` | Develop components locally using Storybook
+`STORIES=path/to/package pnpm storybook` | Develop just one package at a time using Storybook (builds faster!)
+`pnpm commit` | Use commitizen to help you write your conventional commits
```

Comment:
Lol oops this isn't correct :D
Maybe you should change the script name to just be `changeset` (or just nuke this one)

Reply:
Can never seem to get rid of it haha